src/test/java/org/minions/devfund/richard/CountCharactersTest.java

```diff
@@ -0,0 +1,23 @@
+package org.minions.devfund.richard;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Unit Tests for CountCharacters.
+ */
+public class CountCharactersTest {
+
+    /**
+     * Sample test provided by CodeWars.
+     */
+    @Test
+    public void testCodeWars() {
+        final int expectedFour = 4;
+        final int expectedFive = 5;
+
+        assertEquals(expectedFour, CountCharacters.countChar("fizzbuzz", 'z'));
+        assertEquals(expectedFive, CountCharacters.countChar("Fancy fifth fly aloof", 'f'));
```

Comment:
Your implementation is failing for the following scenarios.
```java
        assertEquals(expectedFour, CountCharacters.countChar("fizzbuzz", 'Z'));
        assertEquals(expectedFive, CountCharacters.countChar("Fancy fifth fly aloof", 'F'));
```

Reply:
Ok I am going to do that.